packages/SwingSet/src/vats/comms/controller.js

```diff
@@ -34,7 +34,9 @@ export function deliverToController(
 
     const name = args[0];
     const transmitterID = krefOf(args[1]);
+    assert(transmitterID, 'unexpected "transmitter" arg for addRemote()');
     const setReceiverID = krefOf(args[2]);
+    assert(setReceiverID, 'unexpected "setReceiver" arg for addRemote()');
```

Comment:
Not sure I understand the error text.  Did you mean "missing" instead of "unexpected"?  We _are_ expecting a transmitter arg, after all.  That's what this assert is checking for.

Reply:
Oh, my first commits were made thinking that `krefOf` would return `null` for non-object-references instead of throwing. In that case, we need to assert that `setReceiverID` is non-null tor make sure it wasn't a non-object-reference, like `setReceiverID = 4`, or `setReceiverID = { foo: 'bar' }` (or indeed `undefined`, in which case the arg was probably missing).

After making that commit, I got bolder and added a later commit which makes `krefOf` throw. In *that* case, the `assert` is redundant, but harmless.

By "unexpected" I really should have said "unexpected type of argument", but that got too long :). How about I change it to `bad "setReceiver" arg`? That's probably less ambiguous.